src/plugins/intel_gpu/src/graph/impls/ocl/kv_cache.cpp

```diff
@@ -186,10 +226,14 @@ struct kv_cache_impl : multi_stage_primitive<kv_cache> {
         const auto& desc = instance.get_typed_desc<kv_cache>();
         auto& variable = instance.get_network().get_variable(desc->variable_info.variable_id);
         std::vector<event::ptr> res_events;
+        const auto& impl_param = *instance.get_impl_params();
+
+        if (reorder_stage == static_cast<size_t>(0)) { 
```

Comment:
Nitpick: this is too explicit, could be replaced with:
```suggestion
        if (!reorder_stage) { 
```

Reply:
IMO, this code is difficult to understand. What about adding explicit flag in `impl_param`?